motoman_msgs/srv/ReadGroupIO.srv

```diff
@@ -0,0 +1,21 @@
+
+# Read (and return) the current value of the Group IO element at 'address'.
+#
+# Addresses are plain, base-10 integers, as used and displayed by the controller
+# (on the teach pendant for instance).
+#
+# There are no restrictions as to which group IO elements can be read, but they
+# have to exist on the controller and be configured correctly.
+#
+# NOTE: many programming languages will parse literals starting with '0' as
+#       octal numbers. Do not add leading zeros to group addresses to avoid
+#       specifying the wrong address to read.
+#
+# Refer also the Yaskawa Motoman documentation on IO addressing and
+# configuration.
+
+uint32 address
+---
+string message
+bool success
+uint32 value
```

Comment:
We could consider changing this to `uint8`, to match the width of the IO groups on the controller.

@ted-miller / @EricMarcil: group IO is always 8bit, isn't it?

Reply:
Friendly ping @ted-miller

Reply:
Yes, a group is always 8 bit. 
There are some operations in Inform and CIO that will use 16 bits. But, these are always considered to use two groups.